docs/ldml/tr35-keyboards.md

```diff
@@ -298,12 +306,85 @@ This mandatory attribute represents the locale of the keyboard using Unicode loc
   …
 </keyboard>
 ```
+
 ```xml
 <keyboard locale="fr-CH-t-k0-android">
```

Comment:
Given the work we've done on the spec, and our goal to facilitate cross-platform keyboards, I find it concerning that our two examples for locale both include a platform name, which would imply non-cross-platform design.

_One_ is fine to show support for older keyboard designs that weren't made with this in mind, but having both of our only two examples here do this is a bit concerning due to the inferences an uninitiated reader may draw.

Reply:
- [ ] Make non platform ids more prominent. 

Yes. This is the balance between leveraging what was written before and starting over. I thought about that while linking in this section. We should prefer non platform ids.